Ability to unload all weapons in inventory via context menu.

Dx74rghs opened the pull request:
Functionality to unload all ammo from available weapons in the backpack.

Dx74rghs committed:
Unload all ammo

Functionality to unload all ammo from available weapons in the backpack.

Xottab-DUTY reviewed (changes requested):
I can't accept this PR as is. This introduces gamedata changes, which we try to keep close to vanilla as possible.

You can do it another way: try to add a context menu entry with string "Unload all". Yes, it's not a hotkey, but this is a good compromise between introducing new features and being close to vanilla as possible.

Also, do not put the translations into original `ui_st_keybinding.xml`, instead use `openxray.xml`.

Dx74rghs committed:
Revert "Unload all ammo"

This reverts commit 6a479ad5689ab1004d691e5dea77b5c46facf2c5.

Dx74rghs committed:
Unload all weapons functionality

Context menu item to unload all weapons in the actor's backpack. Appears only for loaded weapons (same conditions as for single "Unload").

Dx74rghs committed:
translation fix

translation fix for context item
